pkgs/by-name/sa/salmon/fetch-pufferfish.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/scripts/fetchPufferfish.sh b/scripts/fetchPufferfish.sh
+index bf2574e0..42582806 100755
+--- a/scripts/fetchPufferfish.sh
++++ b/scripts/fetchPufferfish.sh
+@@ -1,4 +1,4 @@
+-#!/bin/bash
++#!${bash}/bin/bash
```

Comment:
This doesn't seem right: this is not a nix expression and the `bash` environment variable isn't set, so this likely has no effect.